securedrop_client/message_sync.py

```diff
@@ -29,47 +29,42 @@
 from PyQt5.QtCore import QObject
 from securedrop_client import storage
 from securedrop_client import crypto
-from securedrop_client.models import make_engine
-
-from sqlalchemy.orm import sessionmaker
 
 
 logger = logging.getLogger(__name__)
 
 
 class APISyncObject(QObject):
 
-    def __init__(self, api, home, is_qubes):
+    def __init__(self, api, session, gpg):
```

Comment:
Yeah that was what I was posting about in gitter. However, this does still create a problem that if we pass a reference to this object, it could still be tainted. I think I'm going to roll this back for now so that each has its own session.